packages/big-design/src/components/Worksheet/Row/Row.tsx

```diff
@@ -22,6 +22,7 @@ const InternalRow = <T extends WorksheetItem>({ columns, rowIndex }: RowProps<T>
           columnIndex={columnIndex}
           hash={column.hash}
           key={`${rowIndex}-${columnIndex}`}
+          options={column.type === 'select' ? column.options : []}
```

Comment:
Could we potentially allow `options` to be `undefined`? We default it here https://github.com/bigcommerce/big-design/pull/520/files#diff-89a2998a7d4275373c1d9b534a177558d58446e282bc6c05197f9dead4bd8dd7R20

Reply:
I will change it to be allow `undefined` along the way and let the default handle the scenario of an empty `options` array.